### PR TITLE
only left and inner joins

### PR DESCRIPTION
**Pull request by Nowosad**
@jannes-m @Robinlovelace take a look and merge it if you think that this change is fine

### CODE REVIEW

File: 03-attribute-operations.Rmd [hunk 1]
```diff
@@ -297,7 +297,8 @@ world_continents2 = aggregate(world["pop"], by = ag_var, FUN = sum, na.rm = TRUE
 Combining data from different sources is one of the most common tasks in data preparation. 
 Joins are methods to combine pair of tables based on a shared key variable.
 The **dplyr** package has a set of verbs to easily join `data.frames` - `left_join()`, `right_join()`,  `inner_join()`, `full_join`, `semi_join()` and `anti_join()`.
-The function names follow SQL naming conventions, and @grolemund_r_2016 explain these functions thoroughly in the Relational data chapter in the book R for Data Science .
+
+The function names follow SQL naming conventions, and @grolemund_r_2016 explain these functions thoroughly in the Relational data chapter in the book R for Data Science.
```
Comment (Robinlovelace) [inline 03-attribute-operations.Rmd]: This single sentence paragraph seems a bit lonely. I suggest removing the preceding line break (deleting L300) and tweaking replacing this line with a slighly modified sentence:

```
These function names follow conventions used in the database language SQL, as explained in [Chapter 13](http://r4ds.had.co.nz/relational-data.html) of *R for Data Science* [@grolemund_r_2016].
```

Shorter and clearer I think.

File: 03-attribute-operations.Rmd [hunk 2]
```diff
@@ -332,6 +333,9 @@ Most of the following join examples will have a `sf` object as the first argumen
 However, the reverse order is also possible and will give you back a `data.frame` object.
 This is mostly beyond the scope of this book, but we encourage you to try it.
 
+The next subsections will focus on the two most often used types of joins - left and inner joins.
+We recommend to read @grolemund_r_2016 to learn more about the rest of the joins types.
```
Comment (Robinlovelace) [inline 03-attribute-operations.Rmd]: I would delete this line and replace the preceding line with the following text to make the section flow better and more concise:

```
The next subsections focuses on the commonly used left and inner joins, which use the same syntax as the other join types [@grolemund_r_2016].
```

File: 03-attribute-operations.Rmd [hunk 3]
```diff
@@ -332,6 +333,9 @@ Most of the following join examples will have a `sf` object as the first argumen
...
+
 ### Left joins
 
 The left join is the most often used type of joins.
```
Comment (Robinlovelace) [inline 03-attribute-operations.Rmd]: Suggestion (fixing a plurality typo):

```
Left joins are the most commonly used operation for adding attributes to spatial data. 
```

File: 03-attribute-operations.Rmd [hunk 4]
```diff
@@ -584,7 +531,7 @@ Hint: try to use helper functions, such as `contains` or `starts_with` from **dp
 6. What was the minimum and maximum total population in 2015 in each region? What was the total population in 2015 in each region?
 <!-- Attribute joining -->
 7. Add variables from `us_states_df` to `us_states` and create a new object called `us_states_stats`. What function did you use and why? Which variable is the key in the both datasets? What is the class of a new object?
-8. `us_states_df` has two more variables than `us_states`. How you can find them?
+8. `us_states_df` has two more variables than `us_states`. How you can find them? (hint: try to use the `dplyr::anti_join` function)
```
Comment (Robinlovelace) [inline 03-attribute-operations.Rmd]: That's a good challenge. Just wondering if there's another question we could ask about right joins but nothing comes to mind.

**Comment by Robinlovelace** [comment]
Let me know when you've addressed the issues above and I'll merge this. Thanks!

**Comment by Nowosad** [comment]
I use right joins every few months, but for a very specific purposes.. I will think about some examples for the book (no idea yet).

**Comment by Robinlovelace** [comment]
Cheers.

**Pull request merged**